normandy/settings.py

```diff
@@ -357,6 +357,13 @@ def RAVEN_CONFIG(self):
     AUTOGRAPH_SIGNATURE_MAX_AGE = values.IntegerValue(60 * 60 * 24 * 7)
     AUTOGRAPH_X5U_CACHE_BUST = values.Value(None)
 
+    # Remote Settings settings
+    REMOTE_SETTINGS_ENABLED = values.BooleanValue(True)
+    REMOTE_SETTINGS_URL = values.Value("http://localhost:8888/v1")
+    REMOTE_SETTINGS_USERNAME = values.Value("normandy")
+    REMOTE_SETTINGS_PASSWORD = values.Value("n0rm4ndy")
+    REMOTE_SETTINGS_COLLECTION_ID = values.Value("normandy-recipes")
```

Comment:
For the final version, you'll probably want to leave these as blanks (like `REMOTE_SETTINGS_URL = values.Value()`) since I don't think we can supply reasonable defaults.